app/config.js

```diff
@@ -16,9 +16,16 @@ config.log = {
 
 config.apps = [
   'base',
+  'apps',
   'users'
 ];
 
+config.session = {
+  secret: process.env.COOKIE_SECRET || 'shh-its-a-secret',
```

Comment:
I would probably avoid a default in this case or even throw an exception in case this env. variable is missing (misconfiguration).

PS: This is just a minor observation so feel free to ignore if you reckon is not important, etc...

Reply:
The secret MUST be set explicitly in the environment in deployment, but it's not very important in dev, so the default just makes life easier. All the defaults are (should be) tuned for running in development.